File: app/src/main/java/com/example/pam_project/features/lists/create/CreateListView.java
package com.example.pam_project.features.lists.create;

import com.example.pam_project.features.categories.list.CategoryInformation;

import java.util.List;

public interface CreateListView {

    void bindCategories(final List<CategoryInformation> model);

    void onListInsertedError();

    void onCategoriesReceivedError();
}

File: app/src/main/java/com/example/pam_project/features/lists/create/CreateListView.kt
package com.example.pam_project.features.lists.create

import com.example.pam_project.features.categories.list.CategoryInformation

interface CreateListView {
    fun bindCategories(model: List<CategoryInformation?>?)
    fun onListInsertedError()
    fun onCategoriesReceivedError()
}